src/drivers/pic/pic.zig

```diff
@@ -29,10 +32,11 @@ pub const IRQS = enum {
     Coprocessor,
     PrimaryATAHardDisk,
     SecondaryATAHardDisk,
+    Unknown,
 };
 
-fn get_irq_port_id(irq: IRQS) struct { id: u8, port: cpu.Ports } {
-    var id: u8 = @intFromEnum(irq);
+fn get_irq_port_id(irq: IRQ) struct { id: u8, port: cpu.Ports } {
+    var id: u8 = @intFromEnum(@as(IRQ, irq));
```

Comment:
is `@as` really necessary? (since irq is already typed IRQ)

Reply:
Whoops, just some residue from a test for a more generic function without success :eyes:

Reply:
Done